src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -23,14 +24,19 @@ sealed class ConfigurationFeatureDefinitionProvider : IFeatureDefinitionProvider
 
         private const string FeatureFiltersSectionName = "EnabledFor";
         private const string RequirementTypeKeyword = "RequirementType";
+        private const string FeatureManagementSectionName = "FeatureManagement";
         private readonly IConfiguration _configuration;
         private readonly ConcurrentDictionary<string, FeatureDefinition> _definitions;
         private IDisposable _changeSubscription;
+        private readonly ILogger _logger;
         private int _stale = 0;
 
-        public ConfigurationFeatureDefinitionProvider(IConfiguration configuration)
+        const string ParseValueErrorString = "Invalid setting '{0}' with value '{1}' for feature '{2}'.";
```

Comment:
> ParseValueErrorString

What is this?

Reply:
Oops. It's part of the preview branch https://github.com/microsoft/FeatureManagement-Dotnet/blob/preview/src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs#L30 . Must have left it when changing the target here. Removing now